<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lidar_testClass</class>
 <widget class="QMainWindow" name="Lidar_testClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2260</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lidar_test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current Color Field：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_color"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ChangeColor">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Change</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>911</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current View：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_View">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Top View</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Special View</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_GetCurrentView">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Get Current View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_GetCurrentView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>271</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox_Loop">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>loop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Grid">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>grid</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_ChangeColor_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Adjust PointCloud</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2260</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>INVENT PointCloud Viewer V1.0</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lidar_test.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_ChangeColor</sender>
   <signal>clicked()</signal>
   <receiver>Lidar_testClass</receiver>
   <slot>on_PushButton_ChangeColor_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_View</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Lidar_testClass</receiver>
   <slot>on_ComboBox_View_Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Grid</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Lidar_testClass</receiver>
   <slot>on_CheckBox_Grid_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_GetCurrentView</sender>
   <signal>clicked()</signal>
   <receiver>Lidar_testClass</receiver>
   <slot>on_PushButton_GetCurrentView_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ChangeColor_2</sender>
   <signal>clicked()</signal>
   <receiver>Lidar_testClass</receiver>
   <slot>on_PushButton_AdjustPC_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_PushButton_ChangeColor_Clicked()</slot>
  <slot>on_ComboBox_View_Changed()</slot>
  <slot>on_CheckBox_Grid_stateChanged()</slot>
  <slot>on_PushButton_GetCurrentView_Clicked()</slot>
  <slot>on_PushButton_AdjustPC_Clicked()</slot>
  <slot>on123()</slot>
 </slots>
</ui>
